pkg/render/common/esgateway/esgateway.go

```diff
@@ -0,0 +1,9 @@
+package esgateway
+
+const (
+	EsGatewayElasticServiceName      = "tigera-secure-es-http"
```

Comment:
Nit, Could you please remove the `EsGateway` prefix here. The package name is `esgateway` so `esgateway.EsGateway` is redundant.

Reply:
Nit, please don't shorten `Elasticsearch` to `Elastic`, `Elastic` has a meanings outside of `Elasticsearch` in computing. I'd just say Elasticsearch.